pkg/kubelet/dockertools/manager.go

```
@@ -511,6 +511,9 @@ func (dm *DockerManager) runContainer(pod *api.Pod, container *api.Container, op
 	if len(opts.DNSSearch) > 0 {
 		hc.DNSSearch = opts.DNSSearch
 	}
+	if len(opts.CgroupParent) > 0 {
```

I assume this option will be silently ignored with older versions of docker.